### PR TITLE
README: Clarify the supported version and the solution for failing installation

### PR DESCRIPTION
**Pull request by pongloongyeat**
Use codename instead of version numbering as was with previous tweaks and point to previous tweaks in description (#40).

### CODE REVIEW

**Comment by pongloongyeat** [comment]
Should probably tell users to refer to use [these commands](https://github.com/pantheon-tweaks/pantheon-tweaks/issues/43#issuecomment-720077052) temporarily until eOS6 is official?

**Comment by pongloongyeat** [comment]
Should probably tell users to refer to use [these commands](https://github.com/pantheon-tweaks/pantheon-tweaks/issues/43#issuecomment-720077052) temporarily until eOS6 is official?

**Comment by ryonakano** [comment]
Regarding to the copy, what about something like this?

> A system settings panel for the Pantheon Desktop that lets you easily and safely customize your desktop's appearance.
> 
> Pantheon Tweaks only supports elementary OS **Odin**. For users on elementary OS **Juno** or below, please use [elementary tweaks](https://github.com/elementary-tweaks/elementary-tweaks) instead.

**Comment by ryonakano** [comment]
> Should probably tell users to refer to use [these commands](https://github.com/pantheon-tweaks/pantheon-tweaks/issues/43#issuecomment-720077052) temporarily until eOS6 is official?

Yeah, I think we may want to write down these commands in the README for now until elementary OS devs fixes the issue on adding PPAs. @pongloongyeat Would you also perform this change in this pull request?

**Comment by pongloongyeat** [comment]
@ryonakano I've added "**Odin** and above" since I'm guessing this tweaks tool will be maintained for upper versions of Pantheon.

**Comment by ryonakano** [comment]
Hmm, I think users may not find the notice because it's written after the current command that is not working. I prefer supersede the current command with the command that works, like this:

```diff
Add the PPA of Pantheon Tweaks and then install it:

- sudo add-apt-repository ppa:philip.scott/pantheon-tweaks
+ sudo apt-key adv --keyserver keyserver.ubuntu.com --recv c42d52715a84c7d0d02fc740c1d89326b1c71ab9
+ echo -e "deb http://ppa.launchpad.net/philip.scott/pantheon-tweaks/ubuntu focal main\ndeb-src http://ppa.launchpad.net/philip.scott/pantheon-tweaks/ubuntu focal main" | sudo tee /etc/apt/sources.list.d/pantheon-tweaks.list
+ sudo apt update
sudo apt install pantheon-tweaks

Open System Settings and there should be a new Plug named "Tweaks". 

- **Note:** if you are currently on Odin daily builds, follow [this](https://github.com/pantheon-tweaks/pantheon-tweaks/issues/43#issuecomment-720077052) to install Pantheon Tweaks instead.
```

> I've added "**Odin** and above" since I'm guessing this tweaks tool will be maintained for upper versions of Pantheon.

I would like you to keep it just "**Odin**" because it's not sure if Pantheon Tweaks works on future versions of elementary OS until they are released (at least as a daily build). We may not want to say "supported" until we can confirm it really works on them. We would keep it maintained through the future though.

**Comment by pongloongyeat** [comment]
Wouldn't they notice the note when they realise the command fails? Also, wouldn't this require additional updating when official images of eOS6 are out?

**Comment by ryonakano** [comment]
> Wouldn't they notice the note when they realise the command fails?

Yeah, the issue was already reported at https://github.com/elementary/os-patches/issues/136 but I'm not sure they care about it because they don't recommend adding PPAs or installing apps from there and recommend use of curated or *flatpaked* apps.

> Also, wouldn't this require additional updating when official images of eOS6 are out?

Yeah, but even if we choose to add the "notice" section instead we need to remove it when Odin is released. We need some modification in README later anyways so it doesn't matter :wink:

**Comment by pongloongyeat** [comment]
@ryonakano its done. Sorry for the late changes, flooded with exams.

File: README.md [hunk 1]
```diff
@@ -14,11 +16,13 @@ sudo apt install software-properties-common
...
 ```
-sudo add-apt-repository ppa:philip.scott/pantheon-tweaks
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv c42d52715a84c7d0d02fc740c1d89326b1c71ab9
+echo -e "deb http://ppa.launchpad.net/philip.scott/pantheon-tweaks/ubuntu focal main\ndeb-src http://ppa.launchpad.net/philip.scott/pantheon-tweaks/ubuntu focal main" | sudo tee /etc/apt/sources.list.d/pantheon-tweaks.list
+sudo apt update
 sudo apt install pantheon-tweaks
 ```
 
-Open System Settings and there should be a new Plug named "Tweaks".
+Open System Settings and there should be a new Plug named "Tweaks".
```
Comment (ryonakano) [inline README.md:25]:
```suggestion
Open System Settings and there should be a new Plug named "Tweaks".
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Pantheon Tweaks
-A system settings panel for the Pantheon Desktop and elementary OS **6.0** that lets you easily and safely customise your desktop's appearance 
+A system settings panel for the Pantheon Desktop that lets you easily and safely customise your desktop's appearance.
+
+Pantheon Tweaks is currently only supported on elementary OS **Odin**. For users on elementary OS **Juno** or below, please instead use [elementary tweaks](https://github.com/elementary-tweaks/elementary-tweaks).
```
Comment (ryonakano) [inline README.md:4]:
```suggestion
Pantheon Tweaks is currently only supported on elementary OS **Odin**. For users on elementary OS **Juno** or below, please instead use [elementary Tweaks](https://github.com/elementary-tweaks/elementary-tweaks).
```

**Review by ryonakano — CHANGES_REQUESTED** [review]
Final small changes request from me and it's ready to go

**Review by ryonakano — APPROVED** [review]
LGTM, thank you for your contribution! :tada:

**Pull request merged**